docker-compose-localstack.yml

```diff
@@ -0,0 +1,21 @@
+version: '2.1'
+
+services:
+  localstack:
+    container_name: "${LOCALSTACK_DOCKER_NAME-localstack_main}"
+    image: localstack/localstack
+    network_mode: bridge
+    ports:
+      - "4566:4566"
+      - "4571:4571"
+    environment:
+      - LOCALSTACK_SERVICES=${LOCALSTACK_SERVICES-sns}
+      - LOCALSTACK_DEFAULT_REGION=${LOCALSTACK_DEFAUL_REGION-eu-west-1}
```

Comment:
🙋‍♂️ I'm not sure of the implications, but we default to `eu-west-2` (London) due to data handling related to individuals

Reply:
no implications for localstack, but i'll change for clarity.